eloquent-js/chap14/chap14q3.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Eloquent JS Chapter 14 Q3</title>
+    <style>
+        body {
+            min-height: 200px;
+        }
+
+        img {
+            height: 30px;
+
+        }
+    </style>
+</head>
+<body>
+    <img src="img/cat.png" id="cat" style="position: absolute">
```

Comment:
If you have a style tag in the file, is it intentional to define inline styles instead of using the global styling?